src/style.css

```diff
@@ -0,0 +1,84 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 800;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  padding-top: 12px;
+}
+
+.form {
+  margin-bottom: 20px;
+  position: relative;
+}
+
+.search {
+  font-family: "Avenir", sans-serif;
+  display: block;
+  width: 100%;
+  background: #fff;
+  border-radius: 4px;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  color: #3d4e61;
+  padding: 23px 62px;
+  font-size: 16px;
+  line-height: 22px;
+  font-weight: 300;
+}
+
+.search--small {
+  padding: 10px 10px 11px 33px;
+  font-size: 14px;
+  line-height: 19px;
+}
+
+.search::placeholder {
+  font-size: 16px;
+  line-height: 22px;
+  font-family: "Avenir", sans-serif;
+  color: #3d4e61;
+  font-weight: 300;
+}
+
+.search:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.search:focus {
+  font-weight: 800;
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  outline: none;
+}
+
+.search--small::placeholder {
+  font-size: 14px;
+  line-height: 19px;
+  filter: drop-shadow(0 4px 4px rgba(0, 0, 0, 0.25));
```

Comment:
This styles should be set for input text, not for placeholder, it may break your tests